base/applications/control/control.c

```diff
@@ -99,6 +138,28 @@ wWinMain(HINSTANCE hInstance,
     else if (!_wcsicmp(lpCmdLine, L"userpasswords"))   return RunControlPanel(L"nusrmgr.cpl");       /* Graphical User Account Manager */
     else if (!_wcsicmp(lpCmdLine, L"userpasswords2"))  return RUNDLL(L"netplwiz.dll,UsersRunDll");   /* Dialog based advanced User Account Manager */
 
+    /* https://learn.microsoft.com/en-us/windows/win32/shell/executing-control-panel-items#windows-vista-canonical-names */
+    argv = CommandLineToArgvW(lpCmdLine, &argc);
+    if (argv)
+    {
+        UINT argi = 0;
+        HRESULT hr = -1;
+        if (argc >= 2 && IsSwitch(L"name", argv[argi + 0]))
```

Comment:
This makes them positional, is this expected?

Reply:
Yes, already asked and answered above. Windows only supports /name before /page.

Reply:
ok, sorry :)